demo/tests/test_exception_handler.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
```

Comment:
Nitpick: The first import must be after the module's docstrings.

Reply:
fixed and pushed.